src/lib.rs

```diff
@@ -19,14 +21,12 @@ macro_rules! defer {
 
 /// `Guard` is a scope guard that may own a protected value.
 ///
-/// If you place a guard value in a local variable, its destructor will
+/// If you place a guard in a local variable, the closure will
 /// run regardless how you leave the function — regular return or panic
 /// (barring abnormal incidents like aborts; so as long as destructors run).
 ///
-/// The guard's closure will be called with a mut ref to the held value
-/// in the destructor. It's called only once.
```

Comment:
I have now worked the sentences into the other paragraphs.

I also changed the crate root doc, which I didn't realize was committed, and put it in the readme and crate description.